v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -33,35 +36,35 @@ core FRAME pallets.
 
 This guide will show you how you can add the
 [Nicks pallet](/rustdocs/latest/pallet_nicks/index.html). You can follow
-similar patterns to add additional FRAME pallets to your runtime, however you should note that each
+similar pattern to add additional FRAME pallets to your runtime. However you should note that each
 pallet is a little different in terms of the specific configuration settings needed to use it
 correctly. This tutorial will help you understand what you'll need to consider when adding a new
 pallet to your FRAME runtime.
 
 <Message
   type={`green`}
   title={`We're here to help`}
-  text={`If you run into an issue on this tutorial, **we are here to help!** You can
+  text={`If you run into issues on this tutorial, **we are here to help!** You can
   [ask a question on Stack Overflow](https://stackoverflow.com/questions/tagged/substrate) and use the
   \`substrate\` tag or contact us on [Element](https://matrix.to/#/#substrate-technical:matrix.org).`}
 />
 
-You should already have [latest version of the Node Template](https://github.com/substrate-developer-hub/substrate-node-template/tree/latest) compiled on your computer from when you completed the
+You should already have the [latest version of the Node Template](https://github.com/substrate-developer-hub/substrate-node-template/tree/latest) compiled on your computer from when you completed the
 [Create Your First Substrate Chain](/tutorials/v3/create-your-first-substrate-chain) tutorial. If
 you do not, please complete it first.
 
 <Message
   type={`yellow`}
   title={`Information`}
   text=
-      "Experienced developers who truly prefer to skip those tutorials may install the Node Template
+      "Experienced developers who truly prefer to skip that tutorial may install the Node Template
```

Comment:
```suggestion
      "If you skipped the first blockchain tutorial, you should install the node template now.
```
I don't think there's any value in qualifying developers as experienced (the relevant detail is whether they did the tutorial or did not). No need for the adverb "truly" and "may" implies permission. In this case, I think we are simply saying that the node template is a prerequisite for this tutorial to work as written (and could be in a Before you begin section).